### DISCUSSION TITLE
How should I update the framework I added?

### DISCUSSION THREAD

**Discussion by ragokan** [the-benchmarker/web-frameworks#6810] [Q&A] [answered]
So, I added my framework previous week. Now, I am doing improvements on performance and releasing new versions. What should I do to update? 
- Create a new pull request or is it handled automatically?

**Comment by waghanza** [comment, marked as answer]
There is two cases :
+ The implementation is not correct => you need to open a PR
+ You create a new release => A PR will be made by `renovatebot

**Reply by ragokan** [reply]
Thank you so much!
